doc/developer-guides/hld/hv-rdt.rst

```diff
@@ -0,0 +1,54 @@
+.. _hv_rdt:
+
+RDT Allocation Feature Supported by Hypervisor
+##############################################
+
+The hypervisor allows to use RDT (Resource Director Technology) allocation features to optimize performances of VMs. There are 2 sub-features: CAT (Cache Allocation Technology) and MBA(Memory Bandwidth Allocation), CAT is for cache resources and MBA is for memory bandwidth resources. The RDT in hypervisor is integrated with VM-Configuration feature, the resources consumed by each VM can be configured and restricted at compile time.
+
+CAT High-Level Design
+*********************
+
+Introduction of CAT capabilities
+===============================
+
+On a platform which support CAT, each CPU can mask last-level-cache (LLC) with a cache mask, the masked cache ways cannot be evicted by this CPU. In terms of SDM, CAT abilities are enumerated via CPUID, and configured via MSR registers, these are:
```

Comment:
On a platform which supports CAT
"In terms of SDM, "  please specify the sections in SDM.
"CAT abilities " -> "CAT capabilities"

Reply:
fixed 'supports', 
add " please see chapter 17, volume 3"
fixed "capabilities"